legacy/hogedit/HogEditDoc.cpp

```diff
@@ -738,7 +738,7 @@ FILE *CHogEditDoc::FindFileInHog(char *hog_fname,tHogFileEntry *table_entry)
 	while (pos!=NULL)
 	{
 		entry = Library.filelist.GetNext(pos);
-		if (!strcmpi(entry.name, table_entry->name)) {
```

Comment:
don't replace spaces with tabs (wrong IDE config?)

Reply:
You got it backwards, this file was never reformatted.

Reply:
woops I'm stupid